dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java

```diff
@@ -1344,6 +1339,31 @@ public String execute(final Jedis client, final ConnectionContext state) throws
     }
 
     @Override
+    public String set(final String key, final String value, final String nxxx, final String expx, final long time) {
+        return d_set(key, value, nxxx, expx, time).getResult();
+    }
+    
+
+    public OperationResult<String> d_set(final String key, final String value, final String nxxx, final String expx, final long time) {
+        if (CompressionStrategy.NONE == connPool.getConfiguration().getCompressionStrategy()) {
+            return connPool.executeWithFailover(new BaseKeyOperation<String>(key, OpName.SET) {
+                @Override
+                public String execute(Jedis client, ConnectionContext state) throws DynoException {
+                    return client.set(key, value, nxxx, expx, time);
+                }
+            });
+        } else {
+            return connPool.executeWithFailover(new CompressionValueOperation<String>(key, OpName.SET) {
+                @Override
+                public String execute(final Jedis client, final ConnectionContext state) throws DynoException {
+                    return client.set(key, compressValue(value, state), nxxx, expx, time);
```

Comment:
We need to check if dynomite has the appropriate parsing logic in place for this command.